org.notepad_plus_plus.NotepadPlusPlus.yml

```diff
@@ -0,0 +1,132 @@
+app-id: org.notepad_plus_plus.NotepadPlusPlus
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: npp
+default-branch: stable
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
+   # only 64-bit is supported
+  - --env=WINEARCH=win64
+  # disable wine gecko and mono
+  - --env=WINEDLLOVERRIDES=mscoree,mshtml=
+
+modules:
+  - name: wine64
+    builddir: true
+    # https://wiki.winehq.org/Building_Wine#General_Compiling_Hints
+    config-opts:
+      - --enable-win64
+      - --disable-win16
+      - --disable-tests
+      # https://wiki.gentoo.org/wiki/Wine#USE_flags
+      - --without-alsa
+      - --without-oss
+      - --without-pulse
+      - --without-gecko
+      - --without-gphoto2
+      - --without-gstreamer
+      - --without-mp3
+      - --without-v4l
+      - --without-mono
+      - --without-test
+    cleanup:
+      - "*.a"
+      - "*.la"
+      - /include
+      - /lib/pkgconfig
+      - /lib64/pkgconfig
+      - /lib/cmake
+      - /lib64/cmake
+      - /share/man
+      - /bin/function_grep.pl
+      - /bin/widl
+      - /bin/winebuild
+      - /bin/winecpp
+      - /bin/winedump
+      - /bin/wineg++
+      - /bin/winegcc
+      - /bin/winemaker
+      - /bin/wmc
+      - /bin/wrc
+      - /lib/wine/*.def
+      - /lib64/wine/*.def
+    post-install:
+      - install -Dm 644 ../COPYING.LIB ${FLATPAK_DEST}/share/licenses/wine/LICENSE
+    sources:
+      - type: archive
+        url: https://dl.winehq.org/wine/source/5.0/wine-5.0.2.tar.xz
+        sha256: c2c284f470874b35228327c3972bc29c3a9d8d98abd71dbf81c288b8642becbc
+        size: 22484932
+    only-arches:
+      - x86_64
+
+  # it includes wrestool, used to extract icons from exe files
+  - name: icoutils
+    sources:
+      - type: archive
+        url: https://savannah.nongnu.org/download/icoutils/icoutils-0.32.3.tar.bz2
+        sha256: 17abe02d043a253b68b47e3af69c9fc755b895db68fdc8811786125df564c6e0
+    post-install:
+      - install -Dm 644 COPYING /${FLATPAK_DEST}/share/licenses/icoutils/LICENSE
+
+  - name: npp
+    buildsystem: simple
+    sources:
+      - type: file
+        path: org.notepad_plus_plus.NotepadPlusPlus.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html